### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ColorContrastAnalyser:Ratio for the text"Azureml" is less than required(4.5:1)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4010]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10

**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4.Navigate to "Azureml" text present 
![image](https://user-images.githubusercontent.com/39204909/45818317-f4d33e80-bcfe-11e8-8c94-a23aa246272d.png)

**Actual**
Ratio for the text"Azureml" is less than required(4.5:1)
**Expected**
Ratio for the text"Azureml" should be more than the ratio required(4.5:1)

User Impact
Low vision user will not be aabke to view the content .
**Attachment:**
![bugattachment5](https://user-images.githubusercontent.com/39204909/45818228-b8074780-bcfe-11e8-99d2-6bbf0e9ee418.png)

**Comment by takluyver** [comment]
From a comment on #4678, this seems to have been improved somewhere along the way, and the contrast ratio between the black text and the light grey background is now ~18.

The contrast between the light grey background of inline code and the white of the larger background is much smaller, but I don't believe distinguishing the background is important for understanding, so we'll keep that as is.
